internal/eval_hub/runtimes/shared/logs.go

```
@@ -0,0 +1,90 @@
+	)
+	for offset > 0 && countLogLines(remainder) <= n {
+		readSize := int64(tailReadBlockSize)
+		if readSize > offset {
+			readSize = offset
+		}
+		offset -= readSize
+		buf := make([]byte, readSize)
+		if _, err := f.ReadAt(buf, offset); err != nil {
+			return "", err
+		}
+		remainder = append(buf, remainder...)
```

golangci-lint (2.12.2)

[error] append to slice `buf` with non-zero initialized length

(makezero)